src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo2.components.lowest-price
+  (:require [quo.design-system.colors :as colors]
+            [quo.react-native :as rn]
+            [re-frame.core :as re-frame]))
+
+(defn lowest-price
+  [{:keys [top-value
+           bottom-value
+           margin-top
+           top-value-text-color
+           top-value-bg-color
+           bottom-value-bg-color
+           bottom-value-text-color]}]
+  (let [{:keys [height width]} @(re-frame/subscribe [:dimensions/window])]
+    [rn/view {:style {:display :flex
+                      :flex-direction :column
+                      :width "100%"
+                      :margin-top (int margin-top)
+                      :justify-content :center}}
+     [rn/view {:style {:overflow :hidden}}
+      [rn/view {:style {:border-width 2
+                        :border-color (:text-01 @colors/theme)
+                        :border-radius 1
+                        :margin -2
+                        :top (* 0.017 height)
```

Comment:
could you elaborate, this looks like a magic, components should depend on screen dimensions for sure

Reply:
I tried to optimize how it looks to get the desired look by making it's value a ratio of the height .. To make it responsive

Reply:
are there design requirements for this ? i don't think it should be responsive

Reply:
Used flex layout
Thanks 🙇🏻